dfastmi/gui/dialog_utils.py

```diff
@@ -247,3 +250,53 @@ def gui_text(
     except KeyError:
         application_setting = cstr[0]
     return application_setting
+
+
+def open_pdf_windows(pdf_path : str):
+    pdf_reader_path = get_default_pdf_reader_windows()
+    if pdf_reader_path:
+        try:
+            # Ensure the path is absolute
+            pdf_path = os.path.abspath(pdf_path)
+
+            # Open the PDF file with the default PDF reader in a non-blocking way
+            subprocess.Popen([pdf_reader_path, pdf_path], close_fds=True)
+        except (SystemExit, KeyboardInterrupt) as exception:
+            print(f"Failed to open the PDF file: {exception}")
+            raise exception
+        except Exception as exception:
+            print(f"Failed to open the PDF file: {exception}")
+
+
+def get_default_pdf_reader_windows():
```

Comment:
Private method
```suggestion
def _get_default_pdf_reader_windows():
```

Also update the method call

Reply:
yes good idea